### PR TITLE
Edits and Myrack changes

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-4d90c522b6/how-to/). :bow:

**Comment by cyrichardson** [comment]
@catlook Suggestions are here: 
[cloud-dns-faq_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/1807935/cloud-dns-faq_cyrEdits.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-292835552b/how-to/). :bow:

File: content/cloud-dns/cloud-dns-faq.md [hunk 1]
```diff
@@ -15,56 +15,77 @@ product_url: cloud-dns
 
 #### What is a domain name system (DNS)?
 
-The Domain Name System (DNS) is a system by which internet domain name-to-address and address-to-name resolutions are determined. All domains and their components, such as mail servers, utilize DNS to resolve to the appropriate locations. For example, DNS is used to turn 'www.rackspace.com' into the computer addressable IP address '207.97.209.147'.
+The Domain Name System (DNS) determines internet domain name-to-address
```
Comment (svetkars) [inline content/cloud-dns/cloud-dns-faq.md]: "Internet"?

File: content/cloud-dns/cloud-dns-faq.md [hunk 2]
```diff
@@ -93,15 +116,23 @@ The Cloud DNS API currently supports the following record types:
...
+`SPF` `RR` type as defined in [RFC 4408](http://tools.ietf.org/html/rfc4408).
 
-For more information about supported record types, see the [Cloud DNS Developer Guide](https://developer.rackspace.com/docs/cloud-dns/v1/developer-guide/).
+For more information about supported record types, see the
+[Cloud DNS Developer Guide](https://developer.rackspace.com/docs/cloud-dns/v1/developer-guide/).
 
 #### Can I import and export domains?
 
-Yes, via API. You can import a domain from external providers using a valid bind9-formatted zone file. Similarly, you can export their domain to a bind9-formatted file. Currently, you will not be able to export zone files from MyRackspace to Cloud DNS and should contact Support to request a domain transfer from dedicated resources to Cloud resources.
+Yes, you can do this by using API. You can import a domain from external providers by using a
```
Comment (svetkars) [inline content/cloud-dns/cloud-dns-faq.md]: the API?

Comment (catlook) [reply content/cloud-dns/cloud-dns-faq.md]: nice!

File: content/cloud-dns/cloud-dns-faq.md (hunk 1 above)
Comment (catlook) [reply content/cloud-dns/cloud-dns-faq.md]: don't think "internet" is capped in this case.  Will see what style guide says!

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-cc0386f962/how-to/)

**Comment by svetkars** [comment]
LGTM!

**Pull request merged**